book/chapter-7/README.md

```diff
@@ -133,7 +134,7 @@ class App extends React.Component {
 };
 ```
 
-Because of this change the `Switcher` becomes really simple. We don't need the internal state and the component may be written as a stateless function.
+เนื่องจากการเปลี่ยนแปลงนี้ การทำงานใน `Switcher` จะง่ายมาก เราไม่จำเป็นต้องมีสถานะภายใน และคอมโพเนนท์นี้อาจจะเขียนในรูปของฟังก์ช้่นแบบ `stateless` ได้อีกด้วย
```

Comment:
คอมโพเนนท์นี้อาจจะเขียนในรูปของฟังก์ช้่นแบบ `stateless` --> Component นี้อาจเขียนได้ในรูปแบบของฟังก์ชัน ที่เรียกกันว่า stateless function

Reply:
Fixed https://github.com/reactbkk/react-in-patterns-th/pull/29/commits/6c67319512e256a39557b99785ba768499f3a5f9